modules/console/pages/ui/add-license.adoc

```diff
@@ -0,0 +1,58 @@
+= Manage Enterprise Edition Licenses through Redpanda Console
+:description: Learn how to manage Enterprise Edition licenses in Redpanda Console.
+
+You add, update and check your xref:get-started:licensing/overview.adoc#console[Enterprise Edition license] for both Redpanda and Redpanda Console directly through the Redpanda Console UI.
+
+== Prerequisites
+
+- You must have an Enterprise Edition license. If you don't have a license yet, https://www.redpanda.com/contact[request one^].
++
+If Redpanda Console has enterprise features enabled and it cannot find a valid license either locally or in the connected Redpanda cluster, it shuts down.
+- Redpanda Console must be xref:console:config/connect-to-redpanda.adoc[connected to a Redpanda cluster].
+- Redpanda Console must be xref:console:config/connect-to-redpanda.adoc#admin[configured to connect to the Redpanda Admin API].
+
+TIP: If you don't have Redpanda Console connected to a Redpanda cluster, you can xref:console:config/enterprise-license.adoc[configure Redpanda Console to load the license key from its local configuration].
+
+== Check the license status in Redpanda Console
+
+You can check the expiration date of a license on the Cluster Overview page in Redpanda Console, under the Licensing section.
+
+If the license is due to expire within 30 days, a warning banner is displayed on all pages of Redpanda Console.
+
+See also: xref:get-started:licensing/monitor-license-status.adoc[].
+
+== Upload a new license
+
+When a new license is uploaded through Redpanda Console, it is replicated across the cluster and stored persistently in Redpanda's internal metadata, ensuring it is retained across restarts.
+
+NOTE: Do not use Redpanda Console to upload a new license in Kubernetes. Uploading the license through Redpanda Console does not persist the license in Kubernetes resources such as Secrets or manifests. As a result, the license is lost during Pod restarts, scaling, or redeployments. Instead, use Kubernetes secrets, Helm values, or manifest files to ensure automatic and persistent license management. See xref:get-started:licensing/add-license-redpanda/kubernetes.adoc[].
```

Comment:
Thanks @pgellert -Redpanda Console is calling the Admin API `PUT /v1/features/license`.  I think I should clarify:

```suggestion
NOTE: Do not use Redpanda Console to upload a new license in Kubernetes. When you upload the license through Redpanda Console, it does not persist the license to Kubernetes resources such as Secrets or manifests. As a result, the new license is reset to the original in the Kubernetes resources after upgrades and redeployments. Instead, use Kubernetes secrets, Helm values, or manifest files to ensure automatic and persistent license management. See xref:get-started:licensing/add-license-redpanda/kubernetes.adoc[].
```

This is where I need some clarification from @chrisseto - our post upgrade/install job in the Helm chart resets the license and some other cluster config settings. So I believe we'd want to use the same advice we do cluster configs: to avoid using `rpk cluster` and instead rely on the Helm values (Kubernetes resources).

https://github.com/redpanda-data/helm-charts/blob/56e8cc7a5e8a34ed1a27c8073490e76493ecda86/charts/redpanda/post_install_upgrade_job.go#L136C6-L166C8

Reply:
@pgellert is it expected to set the license again after upgrading Redpanda? Or since we use rolling upgrades should the license be replicated onto the upgraded brokers?

Reply:
Why is a license "lost"? Does the operator require & enforce a license too? If you upload the license through Console the license should still be persisted in Redpanda, so pod restarts shouldn*t matter unless we experience complete data loss.

I see how it's a recommended way, but I'm probably missing why it's crucial to use K8s resources instead of uploading to Redpanda directly.

Reply:
>is it expected to set the license again after upgrading Redpanda?

As @weeco said, the license is stored persistently in Redpanda so it isn't lost during upgrades.

Reply:
I mean that Kubernetes operates under the principle of declarative configuration. When you update your Redpanda cluster by applying a new manifest or updating Helm values, Kubernetes reconciles the actual state of the cluster with the desired state defined in your configuration files.

As part of this process, an update triggers a post-install/upgrade job, which re-applies the Redpanda cluster configuration, including any settings defined in your manifest or Helm values. This includes the license.

Reply:
I tested this and persistence works fine if you only use Redpanda Console to set the license. However, the license gets overwritten if you upgrade like so:

```
helm upgrade --install redpanda redpanda/redpanda \
  --version 5.9.7 \
  --namespace jake \
  --create-namespace \
  --set external.domain=customredpandadomain.local \
  --set statefulset.initContainers.setDataDirOwnership.enabled=true \
  --set image.repository=docker.redpanda.com/redpandadata/redpanda-nightly \
  --set image.tag=v0.0.0-20241017git7e72855 \
  --set console.image.repository=redpandadata/console-unstable \
  --set console.image.tag=master-73eca43 \
  --set enterprise.license=my license
```

Reply:
I've updated my suggestion to the following:

NOTE: Do not use Redpanda Console to update the license if it's already set in your Kubernetes resources. During upgrades or redeployments, license values in your Kubernetes resources will override the license set using Redpanda Console. For consistent license management, set the license using either Redpanda Console or Kubernetes resources but not both.

Reply:
Yeah that's great, thanks @JakeSCahill